package.json

```diff
@@ -75,6 +76,7 @@
     "@microsoft/signalr-protocol-msgpack": "3.1.0",
     "@nodert-win10-rs4/windows.security.credentials.ui": "^0.4.4",
     "big-integer": "1.6.36",
+    "browser-process-hrtime": "1.0.0",
```

Comment:
You are going to have to update every browser-based project with this new dependency.

Reply:
Thanks for the heads up. I have PRs in for all 4.